apps/admin-gui/src/app/vos/components/expiration-settings/expiration-settings.component.ts

```diff
@@ -341,7 +341,7 @@ export class ExpirationSettingsComponent implements OnInit, OnChanges {
       return null;
     }
 
-    let period = config.specialLoa.toString() + '|';
+    let period = config.specialLoa?.toString() + '|' ?? '';
```

Comment:
If the `config.specialLoa` is not defined this will be evaluated to `undefined|`. However there is a null check in this method so I am not sure what is this trying to achieve.